orderbook/src/fee.rs

```diff
@@ -24,16 +24,31 @@ pub struct EthAdapter<T> {
     weth: H160,
 }
 
+pub struct FeeSubsidyConfiguration {
+    pub fee_discount: f64,
```

Comment:
maybe some docs on what is the difference between discount and factor (also make it bit more clear if this is a discount in eth or a discount in gas units)? Also I was wondering if this should be an enum or if we allow both to be set at the same time we should document which one goest first.

Edit: I see it's been documented on the command line argument. Not sure if it's worthwhile copying the documentation here?

Reply:
I'll add documentation and copy it here.